src/citrine/_session.py

```diff
@@ -42,8 +44,14 @@ def __init__(self,
         self.access_token: Optional[str] = None
         self.access_token_expiration: datetime = datetime.utcnow()
 
+        agent = "python-requests/{} {}/{} citrine-python/{}".format(
+            requests.__version__, platform.python_implementation(), platform.python_version(),
+            cp_version)
```

Comment:
From a hierarchy perspective, it seems like
```suggestion
        agent = "{}/{} python-requests/{} citrine-python/{}".format(
            platform.python_implementation(), 
            platform.python_version(),
            requests.__version__,
            cp_version)
```
would be more consistent.  Optional.